### ISSUE TITLE
Incompability with Into the Omega

### ISSUE THREAD

**Issue by Areorah** [EleC0TroN/ER#18]
Into The Omega Version: 1.1.1-beta
End Rebellion Version: 1.8
MC Version: 1.16.2
Fabric Version: 0.9.6 Build:206

I pointed my failing to launch out the creator of Into the Omega and this was his response: 
https://github.com/Draylar/into-the-omega/issues/15#issuecomment-686582682

The error that I am getting is here:
https://hastebin.com/ucunokoxib.cpp

**Comment by EleC0TroN** [comment]
Yes, the method of worldgen was a quick-fix, sorry for inconvenience. If i`ll continue the End Rebellion version, this will be fixed.
